samples/install-without-bom/pom.xml

```diff
@@ -33,6 +33,16 @@
     </dependency>
     <!-- [END vision_install_without_bom] -->
 
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-vision</artifactId>
```

Comment:
We don't want samples for this http/rest client library here as this library is not published from this repository.

Reply:
I see, so  Should samples related to this http/rest samples stay in the java-docs repo?